gen/definitions/deploy_template.yaml

```diff
@@ -0,0 +1,98 @@
+---
+name: Deploy Template
+rest_endpoint: /dna/intent/api/v2/template-programmer/template/deploy
+no_delete: true
+no_update: true
+no_read: true
```

Comment:
If we cannot read anything, there is also no point in providing a data source -> `no_data_source: true`.

Reply:
done